tests/stdlib/t15835.nim

```diff
@@ -1,4 +1,9 @@
-import json
+discard """
+  targets: "c js"
+"""
+
```

Comment:
we should merge this into `tests/stdlib/tjson` (which also tests for multi targets)

(also, stdlib/ should probably be reserved for tfoo where foo is a stdlib module)

Reply:
make sense

Reply:
=> https://github.com/nim-lang/Nim/pull/17262